bfvmm/include/interface/arch/intel_x64/cpuid.h

```diff
@@ -0,0 +1,108 @@
+//
+// Copyright (C) 2019 Assured Information Security, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
```

Comment:
Can we use something other than 'interface' - maybe uapi or just api?

Reply:
I like uapi.

Reply:
Are we just talking about the name of the folder? I like uapi too (I'm guessing that is short for user api)

Reply:
yeah. I already made the change in my code. I like it. Any other comments on the approach? I have the state stuff almost complete. The VMCS is going to take the longest as the API is pretty big (all of the vCPU set/get functions that are for the VMCS). I don't want to invest too much time if we don't like the approach.

Reply:
I would prefer uapi to interface and implementation to impl everywhere